.github/workflows/release-drafter.yml

```diff
@@ -19,7 +23,13 @@ jobs:
         pull-requests: write
     runs-on: ubuntu-latest
     steps:
-      # Drafts your next Release notes as Pull Requests are merged into the default branch
-      - uses: release-drafter/release-drafter@v5
+      - name: Check branch name
+        id: branch_check
+        #-qE : quiet mode Search for a regular expression pattern in the input text using extended(E) regular expressions
+        run: echo "is_release_branch=$(echo $GITHUB_REF | grep -qE 'refs/heads/(release-|master)' && echo true || echo false)" >> "$GITHUB_ENV"
```

Comment:
That's a strange expression..... what is it doing?

Reply:
it would check whether the value of $GITHUB_REF contains either "refs/heads/release-" or "refs/heads/master." If it does, it sets the is_release_branch environment variable to "true." Otherwise, it sets it to "false." The result is then appended to the $GITHUB_ENV environment, making it available for subsequent steps in the GitHub Actions workflow. I am still unsure if it would match with just `release-**` or does it need `refs/heads/(release-|master)`. So I decided to keep it `refs/heads/(release-|master)` . I should make that change in on:push: branches: as well.